server.js

```diff
@@ -0,0 +1,163 @@
+const cluster = require('cluster')
+const express = require('express')
+const bodyParser = require('body-parser')
+const app = express()
+const fs = require('fs')
+const r = require('rethinkdbdash')()
+const config = require('./config.json')
+
+const cpusLength = require('os').cpus().length
+app.use(bodyParser.json())
+
+// DBL webhooks
+app.post('/dblwebhook', async (req, res) => {
+  if (req.headers.authorization) {
+    if (req.headers.authorization === config.dbl_webhook_secret) {
+      req.body.type === 'upvote' ? await addPocket(req.body.user, 25)
+        : console.log('not an upvote??')
+      res.send({status: 200})
+    } else {
+      res.send({status: 401, error: 'You done gone goofed up auth.'})
```

Comment:
I know you just copied over errors from the previous server, but can we clean them up a bit now?